sklearn/impute/tests/test_knn.py

```diff
@@ -237,7 +220,6 @@ def test_knn_imputer_verify(na):
 
 @pytest.mark.parametrize("na", [np.nan, -1])
 def test_knn_imputer_one_n_neighbors(na):
-
```

Comment:
Every change must be intentional and must come with a semantic with regards to their goal.

Potentially, some file or function could be better structured or presented, but those changes aren't related to the goal of this PR.

Hence, this line removal and the ones bellow must be reverted.

Reply:
Ups, sorry. New version of black did this 🤷‍♂️